include/utils/DetailViewUtilsTest.php

```diff
@@ -252,4 +252,137 @@ public function testgetRelatedListInfoById() {
 		);
 		$this->assertEquals($expected, $relationInfo, 'testgetRelatedListInfoById');
 	}
-}
\ No newline at end of file
+
+	/**
+	 * Method testgetRelatedListInfoById
+	 * @test
+	 */
+	 public function testgetRelatedModulesInfomation() {
+		$currentModule = 'Accounts';
+		$focus = CRMEntity::getInstance($currentModule);
```

Comment:
You do not use the $focus nor the $restrictedRelations variables so eliminate them